cost/cloud_cost_anomaly_alerts/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v2.4
+
+- added a filter for dimensions when user wants to filter by key pair value like Service=AmazonEC2
```

Comment:
Start with capital letter, Added...

Reply:
How about ?

Added a filter for dimensions where the user wishes to filter by key pair values, such as Service=AmazonEC2